pkg/reconciler/apis/apiexport/apiexport_reconcile.go

```diff
@@ -95,16 +95,19 @@ func (c *controller) reconcile(ctx context.Context, apiExport *apisv1alpha1.APIE
 		)
 	}
 
-	// check if any APIBindings are bound to this APIExport. If so, add a virtualworkspaceURL
-	apiBindings, err := c.getAPIBindingsForAPIExport(clusterName, apiExport.Name)
-	if err != nil {
-		return fmt.Errorf("error checking for APIBindings with APIExport %s|%s: %w", clusterName, apiExport.Name, err)
-	}
+	// TODO(sttts): reactivate this with multi-shard support eventually
```

Comment:
Isn't this being replaced by the APIExportEndpointSlice type?

Reply:
yes. cc @fgiloux